tests/e2e_tests/pages/find.py

```diff
@@ -0,0 +1,87 @@
+import tempfile
+from typing import Literal
+
+from tests.e2e_tests.pages import BasePage
+
+
+class FindRequestDataPage(BasePage):
+    def navigate(self):
+        self.page.goto("http://find-monitoring-data.levellingup.gov.localhost:4001/download")
+
+    def reveal_funds(self):
+        if self.page.get_by_label("Filter by fund , Show this section").is_visible():
+            self.page.get_by_label("Filter by fund , Show this section").click()
+
+    def reveal_regions(self):
+        if self.page.get_by_label("Filter by region , Show this section").is_visible():
+            self.page.get_by_label("Filter by region , Show this section").click()
+
+    def reveal_organisations(self):
+        if self.page.get_by_label("Filter by funded organisation , Show this section").is_visible():
+            self.page.get_by_label("Filter by funded organisation , Show this section").click()
+
+    def reveal_outcomes(self):
+        if self.page.get_by_label("Filter by outcomes , Show this section").is_visible():
+            self.page.get_by_label("Filter by outcomes , Show this section").click()
+
+    def reveal_returns_period(self):
+        if self.page.get_by_label("Filter by returns period , Show this section").is_visible():
+            self.page.get_by_label("Filter by returns period , Show this section").click()
+
+    def filter_funds(self, *funds: str):
+        self.reveal_funds()
+
+        for fund in funds:
+            self.page.get_by_label(fund).check()
+
+    def filter_regions(self, *regions: str):
+        self.reveal_regions()
+
+        for region in regions:
+            self.page.get_by_label(region).check()
+
+    def filter_organisations(self, *organisations: str):
+        self.reveal_organisations()
+
+        for organisation in organisations:
+            self.page.get_by_label(organisation).check()
+
+    def filter_outcomes(self, *outcomes: str):
+        self.reveal_outcomes()
+
+        for outcome in outcomes:
+            self.page.get_by_label(outcome).check()
+
+    def filter_returns_period(
+        self, from_quarter: Literal[1, 2, 3, 4], from_year: str, to_quarter: Literal[1, 2, 3, 4], to_year: str
+    ):
+        self.page.pause()
```

Comment:
From Googling the effect of this method, suspect this and the one below have been left in by accident?